wpilib/wpilib/analoggyro.py

```diff
@@ -37,7 +37,7 @@ class AnalogGyro(GyroBase):
     kDefaultVoltsPerDegreePerSecond = 0.007
     
     PIDSourceType = PIDSource.PIDSourceType
-
+    gyroHandle = 0
```

Comment:
Spacing, should have at least one line above each function.

Also... this feels a bit odd to me. It's not *wrong*. But. I'd rather see this assigned in the constructor.